ql/ql/src/queries/style/AlertMessage.ql

```diff
@@ -164,6 +176,14 @@ String wrongFlowsPhrase(Select sel, string kind) {
   )
 }
 
+/**
+ * Gets a string element that contains double whitespace.
+ */
+String doubleWhitespace(Select sel) {
+  result = getSelectPart(sel, _) and
+  result.getValue().regexpMatch(".*\\s\\s.*")
```

Comment:
Will the `.*` bits match newlines as well (this usually requires a regex flag), or is this perhaps an impossibility anyway?

Reply:
There are no message with "non-space whitespace" currently, and I'm not sure you can have an unescaped newline in a string literal in the current QL language.  

(I want multi-line string literals though 🙏 ).